### PR TITLE
Issue 10

### LINKED ISSUE #10: New Feature: Support -c and --config with Config File

### LINKED ISSUE #11: break convert_to_html down to sub functions
@WangGithub0 Hi Yumei can I ask you to implement this fix since you added the Markdown feature?

Feedback from professor:
I would probably break the md and html aspects of this function into two smaller functions, and name this `convert_txt` or something:

def convert_to_html(text, type):
   If type == ‘markdown':
     return markdown_to_html(text)
   else
     return text_to_html(text)

That’s just pseudo code, but you get the idea.

### PR DESCRIPTION
**Pull request by WangGithub0**
New Feature: Support -c and --config with Config File
Closes #10 

break convert_to_html down to sub functions
Closes #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +48,9 @@ Command-line tool that converts input .txt files into .html files.
 `python convert.py ./examples`
 
 Output(s) can be found in til folder.
-</br></br>
+
+`python convert.py ./examples -c config.toml`
```
Comment (kliu57) [inline README.md]: Can you add this under a section called "Specify Config File". Under the section please include two things (one how to use, one example) (Follow the same format as the "Specify Stylesheet" section)

File: README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ Command-line tool that converts input .txt files into .html files.
 
 ## Features
 
+- `v0.0.4` additionally accept TOML config file
```
Comment (kliu57) [inline README.md:7]: Under "Installation" section, please change #3 to the following and change the current #3 to #4:

3. Install libraries
`pip install tomlkit`

**Review by kliu57 — CHANGES_REQUESTED** [review]
See changes requested

File: README.md (hunk 2 above)
Comment (WangGithub0) [reply README.md:7]: already changed

**Review by kliu57 — APPROVED** [review]
looks good!

**Pull request merged**